src/lsp_data.rs

```diff
@@ -85,11 +86,18 @@ pub mod ls_util {
         }
     }
 
-    pub fn range_from_vfs_file(_vfs: &Vfs, _fname: &Path) -> Range {
-        // FIXME: todo, endpos must be the end of the document, this is not correct
-
-        let end_pos = Position::new(0, 0);
-        Range{ start: Position::new(0, 0), end: end_pos }
+    pub fn range_from_vfs_file(_vfs: &Vfs, _fname: &Path) -> Result<Range, vfs::Error> {
```

Comment:
Why do you think this is a problem if the `vfs::Error` is exposed here? Any caller of this function must have a handle to a `Vfs` anyway and therefore also likely already has to deal with this error. But maybe the error handling should be rethought anyway.

I didn't know the naming convention. I will change that.